doc/decisions/DR023-Manager-capability-introspection.md

```diff
@@ -0,0 +1,275 @@
+# DR023 Manager capability introspection
+
+- **Status:** Decided
+- **Driver:** @foundrytom
+- **Contributors:** Assorted
+- **Outcome:** OpenAssetIO will support manager behaivour introspection
+  using a `managementPolicy` method parameterized by trait set and
+  access.
+
+## Background
+
+A fundemental design constraint of OpenAssetIO is that neither the host
```

Comment:
[fix] I think worth calling out that this is not a new decision, it is retroactive documentation of a decision from years ago. Just in case someone's going through our decision records in order and thinking they're chronological.

Reply:
Good idea - added [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/1281/commits/b9c2a1e49edcf49682a55759b95b0af605795859#diff-06e67d18b764bbffef66d50e34ca8e45da0121b9747633cd115ca5400f7023af).